--- gdm-48.0/daemon/gdm-local-display-factory.c~	2025-03-17 17:07:15.000000000 +0100
+++ gdm-48.0/daemon/gdm-local-display-factory.c	2025-05-10 01:44:03.026665878 +0200
@@ -347,14 +347,14 @@
                 return NULL;
 #endif
 
-        if (!should_fall_back || fallback_display_server == NULL) {
                 if (display_server_enabled (factory, preferred_display_server))
                       g_ptr_array_add (session_types_array, (gpointer) get_session_type_for_display_server (factory, preferred_display_server));
-        }
 
 #ifdef ENABLE_X11_SUPPORT
+    if (should_fall_back && fallback_display_server != NULL) {
         if (display_server_enabled (factory, fallback_display_server))
                 g_ptr_array_add (session_types_array, (gpointer) get_session_type_for_display_server (factory, fallback_display_server));
+    }
 #endif
 
         if (session_types_array->len == 0)
