diff -urN gdm-2.8.0.2.orig/config/gdm.conf.in gdm-2.8.0.2/config/gdm.conf.in
--- gdm-2.8.0.2.orig/config/gdm.conf.in	2005-08-09 15:50:15.572151616 +0200
+++ gdm-2.8.0.2/config/gdm.conf.in	2005-08-09 15:50:41.737173928 +0200
@@ -71,8 +71,8 @@
 # User and group used for running gdm GUI applicaitons.  By default this
 # is set to user gdm and group gdm.  This user/group should have very
 # limited permissions and access to ony the gdm directories and files.
-User=gdm
-Group=gdm
+User=xdm
+Group=xdm
 # To try to kill all clients started at greeter time or in the Init script.
 # doesn't always work, only if those clients have a window of their own
 #KillInitClients=true
