diff -urN gdm.orig/config/Xsession.in gdm-2.4.4.1/config/Xsession.in
--- gdm.orig/config/Xsession.in	2003-08-18 19:31:08.000000000 +0200
+++ gdm-2.4.4.1/config/Xsession.in	2003-09-19 23:59:30.802517128 +0200
@@ -35,6 +35,10 @@
 test -f /etc/xprofile && . /etc/xprofile
 test -f $HOME/.xprofile && . $HOME/.xprofile
 
+if [ ! -f $HOME/.profile ] && echo $SHELL |grep -q "bash"; then
+    test -f $HOME/.bash_profile && . $HOME/.bash_profile
+fi
+
 # Translation stuff
 if [ -x "@EXPANDED_LIBEXECDIR@/gdmtranslate" ] ; then
   gdmtranslate="@EXPANDED_LIBEXECDIR@/gdmtranslate"
