diff -aurN gdm-2.13.0.7.orig/daemon/server.c gdm-2.13.0.7/daemon/server.c
--- gdm-2.13.0.7.orig/daemon/gdm-server.c	2006-01-19 00:32:50.000000000 +0100
+++ gdm-2.13.0.7/daemon/gdm-server.c	2006-02-05 17:12:06.557522000 +0100
@@ -1150,7 +1150,7 @@
         g_debug ("GdmServer: Opening logfile for server %s", logfile);
 
         VE_IGNORE_EINTR (g_unlink (logfile));
-        VE_IGNORE_EINTR (logfd = open (logfile, O_CREAT|O_TRUNC|O_WRONLY|O_EXCL, 0644));
+        VE_IGNORE_EINTR (logfd = open (logfile, O_CREAT|O_TRUNC|O_WRONLY|O_EXCL, 0640));
 
        if (logfd != -1) {
                 VE_IGNORE_EINTR (dup2 (logfd, 1));
