--- gdm-2.0beta4.orig/config/gdm.conf.in	Tue Sep  7 06:37:16 1999
+++ gdm-2.0beta4/config/gdm.conf.in	Sat Feb 12 23:55:15 2000
@@ -3,7 +3,7 @@
 DefaultPath=/bin:/usr/bin:/usr/bin/X11:/usr/local/bin:@bindir@
 DisplayInitDir=@sysconfdir@/gdm/Init
 Greeter=@bindir@/gdmlogin --disable-sound --disable-crash-dialog
-Group=gdm
+Group=xdm
 HaltCommand=/sbin/shutdown -h now
 KillInitClients=1
 LogDir=@authdir@
@@ -11,10 +11,10 @@
 PostSessionScriptDir=@sysconfdir@/gdm/PostSession/
 PreSessionScriptDir=@sysconfdir@/gdm/PreSession/
 RebootCommand=/sbin/shutdown -r now
-RootPath=/sbin:/usr/sbin:/bin:/usr/bin:/usr/bin/X11:/usr/local/bin:@bindir@
+RootPath=/sbin:/usr/sbin:/bin:/usr/bin:/usr/X11/bin:/usr/local/bin:@bindir@
 ServAuthDir=@authdir@
 SessionDir=@sysconfdir@/gdm/Sessions/
-User=gdm
+User=xdm
 UserAuthDir=
 UserAuthFBDir=/tmp
 UserAuthFile=.Xauthority
@@ -52,7 +52,7 @@
 LocaleFile=@localedir@/locale.alias
 Logo=@pixmapdir@/gnome-logo-large.png
 Quiver=1
-SystemMenu=0
+SystemMenu=1
 Welcome=Welcome to %n
 
 [chooser]
