diff -ru gdm-2.2.5.4/daemon/gdm.c gdm-2.2.5.4-AX/daemon/gdm.c
--- gdm-2.2.5.4/daemon/gdm.c	Wed Dec  5 05:29:09 2001
+++ gdm-2.2.5.4-AX/daemon/gdm.c	Tue Feb  5 01:21:32 2002
@@ -579,31 +579,31 @@
 	    gdm_fail (_("gdm_config_parse: Authdir %s is not a directory. Aborting."), GdmServAuthDir);
     }
 
-    if (statbuf.st_uid != GdmUserId || statbuf.st_gid != GdmGroupId)  {
+    if (statbuf.st_uid != 0 || statbuf.st_gid != GdmGroupId)  {
 	    char *s = g_strdup_printf (_("Server Authorization directory "
 					 "(daemon/ServAuthDir) is set to %s "
 					 "but is not owned by user %s and group "
 					 "%s.  Please correct the ownership or "
 					 "gdm configuration %s and restart "
 					 "gdm."),
-				       GdmServAuthDir, GdmUser, GdmGroup,
+				       GdmServAuthDir, "root", GdmGroup,
 				       GDM_CONFIG_FILE);
 	    gdm_text_message_dialog (s);
 	    gdm_fail (_("gdm_config_parse: Authdir %s is not owned by user %s, group %s. Aborting."), 
-		      GdmServAuthDir, GdmUser, GdmGroup);
+		      GdmServAuthDir, "root", GdmGroup);
     }
 
-    if (statbuf.st_mode != (S_IFDIR|S_IRWXU|S_IRGRP|S_IXGRP))  {
+    if (statbuf.st_mode != (S_IFDIR|S_IRWXU|S_IRWXG))  {
 	    char *s = g_strdup_printf (_("Server Authorization directory "
 					 "(daemon/ServAuthDir) is set to %s "
 					 "but has the wrong permissions, it "
-					 "should have permissions of 0750."
+					 "should have permissions of 0770."
 					 "Please correct the permissions or "
 					 "the gdm configuration %s and "
 					 "restart gdm."),
 				       GdmServAuthDir, GDM_CONFIG_FILE);
 	    gdm_text_message_dialog (s);
-	    gdm_fail (_("gdm_config_parse: Authdir %s has wrong permissions %o. Should be 0750. Aborting."), 
+	    gdm_fail (_("gdm_config_parse: Authdir %s has wrong permissions %o. Should be 0770. Aborting."), 
 		      GdmServAuthDir, statbuf.st_mode);
     }
 
