diff -Nuard gdm-2.2.5.5.orig/docs/gdmconfig/C/gdmconfig-C.omf gdm-2.2.5.5/docs/gdmconfig/C/gdmconfig-C.omf
--- gdm-2.2.5.5.orig/docs/gdmconfig/C/gdmconfig-C.omf	Sun Mar  4 17:24:41 2001
+++ gdm-2.2.5.5/docs/gdmconfig/C/gdmconfig-C.omf	Wed Oct  2 15:08:55 2002
@@ -1,14 +1,22 @@
-<?xml version="1.0" standalone="no"?>
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE omf PUBLIC "-//OMF//DTD Scrollkeeper OMF Variant V1.0//EN" "http://scrollkeeper.sourceforge.net/dtds/scrollkeeper-omf-1.0/scrollkeeper-omf.dtd">
 <omf>
   <resource>
+    <creator/>
     <title>
       The GDM Configurator Manual
     </title>
-    <subject>
-      <category>GNOME|Core Desktop</category>
-    </subject>
+    <date>
+	2002-03-12
+    </date>
+    <version identifier="2.2.5.5" date="2002-03-12"/>
+    <subject category="GNOME|Core Desktop"/>
+    <description/>
+    <type/>
     <format mime="text/sgml"/>
     <identifier url="gdmconfig.sgml"/>
     <language code="C"/>
+    <relation seriesid="6670c0da-d607-11d6-9b66-fbf823ee93f7"/>
+    <rights type="GNU GPL" license.version="2" license="http://www.gnu.org/licenses/gpl.html"/>
   </resource>
 </omf>
--- gdm-2.2.5.5.orig/docs/gdmconfig/eu/gdmconfig-eu.omf	Thu Jan 22 00:44:10 2004
+++ gdm-2.2.5.5/docs/gdmconfig/eu/gdmconfig-eu.omf	Thu Jan 22 00:44:10 2004
@@ -1,14 +1,22 @@
-<?xml version="1.0" standalone="no"?>
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE omf PUBLIC "-//OMF//DTD Scrollkeeper OMF Variant V1.0//EN" "http://scrollkeeper.sourceforge.net/dtds/scrollkeeper-omf-1.0/scrollkeeper-omf.dtd">
 <omf>
   <resource>
+    <creator/>
     <title>
       GDM Konfiguratzailearen eskuliburua
     </title>
-    <subject>
-      <category>GNOME|Nukleo Mahaigaina</category>
-    </subject>
+    <date>
+        2002-03-12
+    </date>
+    <version identifier="2.2.5.5" date="2002-03-12"/>
+    <subject category="GNOME|Nukleo Mahaigaina"/>
+    <description/>
+    <type/>
     <format mime="text/sgml"/>
     <identifier url="gdmconfig.sgml"/>
     <language code="eu"/>
+    <relation seriesid="3a62c3d9-d261-4c8b-95a0-55e7ea914630"/>
+    <rights type="GNU GPL" license.version="2" license="http://www.gnu.org/licenses/gpl.html"/>
   </resource>
 </omf>
