--- gdm-3.6.0/configure.ac.old	2012-10-09 16:20:23.603767954 +0200
+++ gdm-3.6.0/configure.ac	2012-10-09 16:23:22.486132194 +0200
@@ -1273,6 +1273,10 @@ else
    X_SERVER="/usr/bin/X"
 fi
 
+X_PATH="/usr/bin"
+X_SERVER_PATH="/usr/bin"
+X_SERVER="/usr/bin/X"
+
 dnl ---------------------------------------------------------------------------
 dnl - Check for Xnest / Xephyr support
 dnl ---------------------------------------------------------------------------
