--- gdm-2.21.4/data/gdm.conf~	2007-11-19 21:57:34.000000000 +0100
+++ gdm-2.21.4/data/gdm.conf	2007-12-20 00:06:06.245930315 +0100
@@ -31,7 +31,7 @@
           send_member="SetValue"/>
   </policy>
 
-  <policy user="gdm">
+  <policy user="xdm">
     <allow send_interface="org.gnome.DisplayManager.Manager"/>
     <allow send_interface="org.gnome.DisplayManager.Display"/>
     <allow send_destination="org.gnome.DisplayManager"
--- gdm-2.21.4/daemon/gdm-display.c~	2007-11-19 21:57:32.000000000 +0100
+++ gdm-2.21.4/daemon/gdm-display.c	2007-12-20 00:38:21.612916949 +0100
@@ -158,7 +158,7 @@
         g_return_val_if_fail (display->priv->access_file == NULL, FALSE);
 
         error = NULL;
-        access_file = _create_access_file_for_user (display, "gdm", &error);
+        access_file = _create_access_file_for_user (display, "xdm", &error);
 
         if (access_file == NULL) {
             g_critical ("could not create display access file: %s", error->message);
--- gdm-2.21.4/daemon/gdm-simple-slave.c~	2007-11-19 21:57:32.000000000 +0100
+++ gdm-2.21.4/daemon/gdm-simple-slave.c	2007-12-20 00:54:11.047664460 +0100
@@ -579,7 +579,7 @@
         }
 
         /* Run the init script. gdmslave suspends until script has terminated */
-        gdm_slave_run_script (GDM_SLAVE (slave), GDMCONFDIR "/Init", "gdm");
+        gdm_slave_run_script (GDM_SLAVE (slave), GDMCONFDIR "/Init", "xdm");
 
         create_new_session (slave);
 
--- gdm-2.21.4/daemon/gdm-factory-slave.c~	2007-11-19 21:57:32.000000000 +0100
+++ gdm-2.21.4/daemon/gdm-factory-slave.c	2007-12-20 00:55:29.542269705 +0100
@@ -460,7 +460,7 @@
         /* FIXME: send a signal back to the master */
 
         /* Run the init script. gdmslave suspends until script has terminated */
-        gdm_slave_run_script (GDM_SLAVE (slave), GDMCONFDIR "/Init", "gdm");
+        gdm_slave_run_script (GDM_SLAVE (slave), GDMCONFDIR "/Init", "xdm");
 
         slave->priv->greeter_server = gdm_greeter_server_new (display_id);
         g_signal_connect (slave->priv->greeter_server,
--- gdm-2.21.7/daemon/gdm-chooser-server.c~	2008-01-30 20:39:16.000000000 +0100
+++ gdm-2.21.7/daemon/gdm-chooser-server.c	2008-02-16 03:59:56.223027208 +0100
@@ -549,14 +549,14 @@
                                          g_param_spec_string ("user-name",
                                                               "user name",
                                                               "user name",
-                                                              "gdm",
+                                                              "xdm",
                                                               G_PARAM_READWRITE | G_PARAM_CONSTRUCT));
         g_object_class_install_property (object_class,
                                          PROP_GROUP_NAME,
                                          g_param_spec_string ("group-name",
                                                               "group name",
                                                               "group name",
-                                                              "gdm",
+                                                              "xdm",
                                                               G_PARAM_READWRITE | G_PARAM_CONSTRUCT));
         signals [HOSTNAME_SELECTED] =
                 g_signal_new ("hostname-selected",
--- gdm-2.21.7/daemon/gdm-greeter-server.c~	2008-01-31 22:42:02.000000000 +0100
+++ gdm-2.21.7/daemon/gdm-greeter-server.c	2008-02-16 04:01:05.718250959 +0100
@@ -931,14 +931,14 @@
                                          g_param_spec_string ("user-name",
                                                               "user name",
                                                               "user name",
-                                                              "gdm",
+                                                              "xdm",
                                                               G_PARAM_READWRITE | G_PARAM_CONSTRUCT));
         g_object_class_install_property (object_class,
                                          PROP_GROUP_NAME,
                                          g_param_spec_string ("group-name",
                                                               "group name",
                                                               "group name",
-                                                              "gdm",
+                                                              "xdm",
                                                               G_PARAM_READWRITE | G_PARAM_CONSTRUCT));
         signals [BEGIN_VERIFICATION] =
                 g_signal_new ("begin-verification",
--- gdm-2.21.7/daemon/gdm-welcome-session.c~	2008-02-02 00:56:22.000000000 +0100
+++ gdm-2.21.7/daemon/gdm-welcome-session.c	2008-02-16 04:02:03.686015025 +0100
@@ -1093,14 +1093,14 @@
                                          g_param_spec_string ("user-name",
                                                               "user name",
                                                               "user name",
-                                                              "gdm",
+                                                              "xdm",
                                                               G_PARAM_READWRITE | G_PARAM_CONSTRUCT));
         g_object_class_install_property (object_class,
                                          PROP_GROUP_NAME,
                                          g_param_spec_string ("group-name",
                                                               "group name",
                                                               "group name",
-                                                              "gdm",
+                                                              "xdm",
                                                               G_PARAM_READWRITE | G_PARAM_CONSTRUCT));
         g_object_class_install_property (object_class,
                                          PROP_SERVER_ADDRESS,
--- gdm-2.21.7/daemon/gdm-xdmcp-chooser-slave.c~	2008-02-06 17:43:37.000000000 +0100
+++ gdm-2.21.7/daemon/gdm-xdmcp-chooser-slave.c	2008-02-16 04:24:26.290973037 +0100
@@ -190,7 +190,7 @@
         }
 
         /* Run the init script. gdmslave suspends until script has terminated */
-        gdm_slave_run_script (GDM_SLAVE (slave), GDMCONFDIR "/Init", "gdm");
+        gdm_slave_run_script (GDM_SLAVE (slave), GDMCONFDIR "/Init", "xdm");
 
         slave->priv->chooser_server = gdm_chooser_server_new (display_id);
         g_signal_connect (slave->priv->chooser_server,
--- gdm-2.21.7/gui/simple-greeter/gdm-user-manager.c~	2008-02-05 16:33:02.000000000 +0100
+++ gdm-2.21.7/gui/simple-greeter/gdm-user-manager.c	2008-02-16 04:25:41.785784450 +0100
@@ -73,7 +73,7 @@
                                   "uucp",       \
                                   "operator",   \
                                   "nobody",     \
-                                  "gdm",        \
+                                  "xdm",        \
                                   "postgres",   \
                                   "pvm",        \
                                   "rpm",        \
