--- gdm-2.26.1/gui/simple-greeter/gdm-layouts.c~	2009-03-27 15:28:45.000000000 +0100
+++ gdm-2.26.1/gui/simple-greeter/gdm-layouts.c	2009-06-29 23:01:14.014597308 +0200
@@ -52,7 +52,7 @@
                 engine = xkl_engine_get_instance (GDK_DISPLAY ());
                 xkl_engine_backup_names_prop (engine);
                 config_registry = xkl_config_registry_get_instance (engine);
-                xkl_config_registry_load (config_registry);
+                xkl_config_registry_load (config_registry, FALSE);
 
                 initial_config = xkl_config_rec_new ();
                 if (!xkl_config_rec_get_from_backup (initial_config, engine)) {
