--- gdm-2.21.4/data/gdm.conf~	2007-11-19 21:57:34.000000000 +0100
+++ gdm-2.21.4/data/gdm.conf	2007-12-20 00:06:06.245930315 +0100
@@ -31,7 +31,7 @@
           send_member="SetValue"/>
   </policy>
 
-  <policy user="gdm">
+  <policy user="xdm">
     <allow send_interface="org.gnome.DisplayManager.Manager"/>
     <allow send_interface="org.gnome.DisplayManager.Display"/>
     <allow send_destination="org.gnome.DisplayManager"
--- gdm-2.21.4/daemon/gdm-display.c~	2007-11-19 21:57:32.000000000 +0100
+++ gdm-2.21.4/daemon/gdm-display.c	2007-12-20 00:38:21.612916949 +0100
@@ -158,7 +158,7 @@
         g_return_val_if_fail (display->priv->access_file == NULL, FALSE);
 
         error = NULL;
-        access_file = _create_access_file_for_user (display, "gdm", &error);
+        access_file = _create_access_file_for_user (display, "xdm", &error);
 
         if (access_file == NULL) {
             g_critical ("could not create display access file: %s", error->message);
--- gdm-2.21.4/daemon/gdm-simple-slave.c~	2007-11-19 21:57:32.000000000 +0100
+++ gdm-2.21.4/daemon/gdm-simple-slave.c	2007-12-20 00:54:11.047664460 +0100
@@ -579,7 +579,7 @@
         }
 
         /* Run the init script. gdmslave suspends until script has terminated */
-        gdm_slave_run_script (GDM_SLAVE (slave), GDMCONFDIR "/Init", "gdm");
+        gdm_slave_run_script (GDM_SLAVE (slave), GDMCONFDIR "/Init", "xdm");
 
         create_new_session (slave);
 
--- gdm-2.21.4/daemon/gdm-factory-slave.c~	2007-11-19 21:57:32.000000000 +0100
+++ gdm-2.21.4/daemon/gdm-factory-slave.c	2007-12-20 00:55:29.542269705 +0100
@@ -460,7 +460,7 @@
         /* FIXME: send a signal back to the master */
 
         /* Run the init script. gdmslave suspends until script has terminated */
-        gdm_slave_run_script (GDM_SLAVE (slave), GDMCONFDIR "/Init", "gdm");
+        gdm_slave_run_script (GDM_SLAVE (slave), GDMCONFDIR "/Init", "xdm");
 
         slave->priv->greeter_server = gdm_greeter_server_new (display_id);
         g_signal_connect (slave->priv->greeter_server,
--- gdm-2.21.4/daemon/gdm-greeter-server.c~	2007-11-19 21:57:32.000000000 +0100
+++ gdm-2.21.4/daemon/gdm-greeter-server.c	2007-12-20 00:56:56.036325165 +0100
@@ -934,14 +934,14 @@
                                          g_param_spec_string ("user-name",
                                                               "user name",
                                                               "user name",
-                                                              "gdm",
+                                                              "xdm",
                                                               G_PARAM_READWRITE | G_PARAM_CONSTRUCT));
         g_object_class_install_property (object_class,
                                          PROP_GROUP_NAME,
                                          g_param_spec_string ("group-name",
                                                               "group name",
                                                               "group name",
-                                                              "gdm",
+                                                              "xdm",
                                                               G_PARAM_READWRITE | G_PARAM_CONSTRUCT));
         signals [BEGIN_VERIFICATION] =
                 g_signal_new ("begin-verification",
--- gdm-2.21.4/daemon/gdm-greeter-session.c~	2007-11-19 22:53:12.000000000 +0100
+++ gdm-2.21.4/daemon/gdm-greeter-session.c	2007-12-20 00:57:34.033713691 +0100
@@ -973,14 +973,14 @@
                                          g_param_spec_string ("user-name",
                                                               "user name",
                                                               "user name",
-                                                              "gdm",
+                                                              "xdm",
                                                               G_PARAM_READWRITE | G_PARAM_CONSTRUCT));
         g_object_class_install_property (object_class,
                                          PROP_GROUP_NAME,
                                          g_param_spec_string ("group-name",
                                                               "group name",
                                                               "group name",
-                                                              "gdm",
+                                                              "xdm",
                                                               G_PARAM_READWRITE | G_PARAM_CONSTRUCT));
         g_object_class_install_property (object_class,
                                          PROP_SERVER_ADDRESS,
