--- gdm-40.0/data/Xsession.in.orig	2021-03-30 15:27:29.673587600 +0200
+++ gdm-40.0/data/Xsession.in	2021-04-08 06:33:38.688182706 +0200
@@ -45,6 +45,10 @@
 fi
 test -f "$HOME/.xprofile" && . "$HOME/.xprofile"
 
+if [ ! -f $HOME/.profile ] && echo $SHELL |grep -q "bash"; then
+    test -f $HOME/.bash_profile && . $HOME/.bash_profile
+fi
+
 # Translation stuff
 if [ -x "@libexecdir@/gdmtranslate" ] ; then
   gdmtranslate="@libexecdir@/gdmtranslate"
@@ -207,12 +211,21 @@
   fi
 fi
 
-# add ssh-agent if found
-sshagent="`gdmwhich ssh-agent`"
-if [ -n "$sshagent" ] && [ -x "$sshagent" ] && [ -z "$SSH_AUTH_SOCK" ]; then
-    command="$sshagent -- $command"
-elif [ -z "$sshagent" ] ; then
-    echo "$0: ssh-agent not found!"
+# Keep in sync with Xclients from xinitrc-ng 
+syswmfile="/etc/sysconfig/desktop"
+
+# In this file "$wmstyledir" value may be given
+[ -f "$syswmfile" ] && . $syswmfile
+
+if [ -z "$wmstyledir" ]; then
+    # Keep in sync with Xclients from xinitrc-ng 
+    wmstyledir="/etc/sysconfig/wmstyle"
+fi
+
+if [ -x "$HOME/bin/$command.sh" ]; then
+    command="$HOME/bin/$command.sh"
+elif [ -x "$wmstyledir/$command.sh" ]; then
+    command="$wmstyledir/$command.sh"
 fi
 
 echo "$0: Setup done, will execute: $command"
