diff -urN gdm-2.4.1.1.orig/config/gdm.conf.in gdm-2.4.1.1/config/gdm.conf.in
--- gdm-2.4.1.1.orig/config/gdm.conf.in	Tue Jan 14 02:59:56 2003
+++ gdm-2.4.1.1/config/gdm.conf.in	Tue Jan 14 03:01:03 2003
@@ -56,8 +56,8 @@
 # you should create these user and group.  Anyone found running this as
 # someone too privilaged will get a kick in the ass.  This should have
 # access to only the gdm directories and files.
-User=gdm
-Group=gdm
+User=xdm
+Group=xdm
 # To try to kill all clients started at greeter time or in the Init script.
 # doesn't always work, only if those clients have a window of their own
 KillInitClients=true
@@ -157,7 +157,7 @@
 # The 'theme'.  By default we're using the default gtk theme
 # Of course assuming that gtk got installed in the same prefix,
 # if not change this.
-#GtkRC=@EXPANDED_DATADIR@/themes/Default/gtk/gtkrc
+GtkRC=@EXPANDED_DATADIR@/themes/Default/gtk-2.0/gtkrc
 # Maximum size of an icon, larger icons are scaled down
 MaxIconWidth=128
 MaxIconHeight=128
