diff -urN gdm-2.4.2.98.orig/daemon/server.c gdm-2.4.2.98/daemon/server.c
--- gdm-2.4.2.98.orig/daemon/server.c	Thu Jul 24 01:18:19 2003
+++ gdm-2.4.2.98/daemon/server.c	Thu Jul 24 01:20:05 2003
@@ -1035,7 +1035,7 @@
         /* Log all output from spawned programs to a file */
 	logfile = g_strconcat (GdmLogDir, "/", d->name, ".log", NULL);
 	unlink (logfile);
-	logfd = open (logfile, O_CREAT|O_TRUNC|O_WRONLY|O_EXCL, 0644);
+	logfd = open (logfile, O_CREAT|O_TRUNC|O_WRONLY|O_EXCL, 0640);
 
 	if (logfd != -1) {
 		dup2 (logfd, 1);
