diff -aurN gdm-2.13.0.8.orig/config/Makefile.am gdm-2.13.0.8/config/Makefile.am
--- gdm-2.13.0.8.orig/config/Makefile.am	2006-02-13 22:09:09.000000000 +0100
+++ gdm-2.13.0.8/config/Makefile.am	2006-02-14 13:25:25.015381840 +0100
@@ -6,7 +6,7 @@
 instsessdir = $(datadir)/xsessions
 initdir = $(sysconfdir)/gdm/Init
 authdir = $(localstatedir)/gdm
-logdir = $(localstatedir)/log/gdm
+logdir = /var/log/gdm
 gnomercdir = $(sysconfdir)/gdm
 postdir = $(sysconfdir)/gdm/PostSession
 predir = $(sysconfdir)/gdm/PreSession
diff -aurN gdm-2.13.0.8.orig/configure.ac gdm-2.13.0.8/configure.ac
--- gdm-2.13.0.8.orig/configure.ac	2006-02-13 22:09:09.000000000 +0100
+++ gdm-2.13.0.8/configure.ac	2006-02-14 13:23:51.662573624 +0100
@@ -818,7 +818,7 @@
 AC_SUBST(EXPANDED_AUTHDIR)
 AC_DEFINE_UNQUOTED(EXPANDED_AUTHDIR,"$EXPANDED_AUTHDIR")
 
-LOGDIR_TMP="$localstatedir/log/gdm"
+LOGDIR_TMP="/var/log/gdm"
 EXPANDED_LOGDIR=`eval echo $LOGDIR_TMP`
 AC_SUBST(EXPANDED_LOGDIR)
 AC_DEFINE_UNQUOTED(EXPANDED_LOGDIR,"$EXPANDED_LOGDIR")
