--- gdm-2.14.10/config/PostSession.in~	2005-02-03 23:57:45.000000000 +0100
+++ gdm-2.14.10/config/PostSession.in	2006-12-18 17:14:41.000000000 +0100
@@ -21,6 +21,6 @@
 
 SESSREG=`gdmwhich sessreg`
 if [ "x$SESSREG" != "x" ] ; then
-	"$SESSREG" -d -w /var/log/wtmp -u /var/run/utmp -x "$X_SERVERS" -h "$REMOTE_HOST" -l "$DISPLAY" "$USER"
+	"$SESSREG" -d -x "$X_SERVERS" -h "$REMOTE_HOST" -l "$DISPLAY" "$USER"
 fi
 exit 0
--- gdm-2.14.10/config/PreSession.in~	2006-05-12 07:58:21.000000000 +0200
+++ gdm-2.14.10/config/PreSession.in	2006-12-18 17:14:20.000000000 +0100
@@ -73,9 +73,9 @@
 if [ "x$SESSREG" != "x" ] ; then
 	# some output for easy debugging
 	echo "$0: Registering your session with wtmp and utmp"
-	echo "$0: running: $SESSREG -a -w /var/log/wtmp -u /var/run/utmp -x \"$X_SERVERS\" -h \"$REMOTE_HOST\" -l \"$DISPLAY\" \"$USER\""
+	echo "$0: running: $SESSREG -a -x \"$X_SERVERS\" -h \"$REMOTE_HOST\" -l \"$DISPLAY\" \"$USER\""
 
-	exec "$SESSREG" -a -w /var/log/wtmp -u /var/run/utmp -x "$X_SERVERS" -h "$REMOTE_HOST" -l "$DISPLAY" "$USER"
+	exec "$SESSREG" -a -x "$X_SERVERS" -h "$REMOTE_HOST" -l "$DISPLAY" "$USER"
 	# this is not reached
 fi
 
