diff -urN gdm-2.6.0.8.orig/config/gdm.conf.in gdm-2.6.0.8/config/gdm.conf.in
--- gdm-2.6.0.8.orig/config/gdm.conf.in	2005-02-14 23:49:35.000000000 +0100
+++ gdm-2.6.0.8/config/gdm.conf.in	2005-03-22 06:59:10.713437016 +0100
@@ -134,7 +134,7 @@
 # Automatic VT allocation.  Right now only works on Linux.  This way
 # we force X to use specific vts.  turn VTAllocation to false if this
 # is causing problems.
-#FirstVT=7
+FirstVT=9
 #VTAllocation=true
 # Should double login be treated with a warning (and possibility to change
 # vts on linux and freebsd systems for console logins)
