--- gdm-2.0beta2/config/gdm.system-auth	Thu Jun  1 01:13:46 2000
+++ gdm-2.0beta2/config/gdm	Thu Jun  1 01:14:32 2000
@@ -1,8 +1,7 @@
 #%PAM-1.0
-auth       required	/lib/security/pam_pwdb.so shadow nullok
+auth       required	/lib/security/pam_stack.so service=system-auth
 auth       required	/lib/security/pam_nologin.so
-account    required	/lib/security/pam_pwdb.so
+account    required	/lib/security/pam_stack.so service=system-auth
-password   required	/lib/security/pam_cracklib.so
-password   required	/lib/security/pam_pwdb.so shadow nullok use_authtok
-session    required	/lib/security/pam_pwdb.so
+password   required	/lib/security/pam_stack.so service=system-auth
+session    required	/lib/security/pam_stack.so service=system-auth
 session    optional     /lib/security/pam_console.so
