--- gdm-2.0beta4/daemon/server.c.orig	Wed Sep 22 20:30:47 1999
+++ gdm-2.0beta4/daemon/server.c	Thu Apr 27 20:42:27 2000
@@ -191,10 +191,11 @@
     if (d->servpid && kill (d->servpid, 0) < 0) {
 	gdm_debug ("gdm_server_restart: Old server for %s still alive. Killing!", d->name);
 	gdm_server_stop (d);
-	gdm_server_start (d);
-	return;
     }
-    
+
+    gdm_server_start (d);
+
+#if 0    
     /* Create new cookie */
     gdm_auth_secure_display (d);
     setenv ("DISPLAY", d->name, TRUE);
@@ -243,6 +244,7 @@
     
     /* Wait for X server to send ready signal */
     pause();
+#endif
 }
 
 
--- gdm-2.0beta2/daemon/auth.c.sopwith	Thu Sep  9 22:44:40 1999
+++ gdm-2.0beta2/daemon/auth.c	Thu Sep  9 22:46:26 1999
@@ -296,7 +296,7 @@
      * to it. So we better play it safe... */
 
     if (! gdm_file_check ("gdm_auth_user_remove", user, authdir, authfile, 
-			  FALSE, GdmUserMaxFile, GdmRelaxPerms)) {
+			  TRUE, GdmUserMaxFile, GdmRelaxPerms)) {
 	gdm_error (_("gdm_auth_user_remove: Ignoring suspiciously looking cookie file %s"), d->userauth);
 	return; 
     }
