diff -urN gdm-2.4.1.1.orig/config/gdm.conf.in gdm-2.4.1.1/config/gdm.conf.in
--- gdm-2.4.1.1.orig/config/gdm.conf.in	Tue Jan 14 02:59:56 2003
+++ gdm-2.4.1.1/config/gdm.conf.in	Tue Jan 14 03:01:03 2003
@@ -56,8 +56,8 @@
 # User and group used for running gdm GUI applicaitons.  By default this
 # is set to user gdm and group gdm.  This user/group should have very
 # limited permissions and access to ony the gdm directories and files.
-User=gdm
-Group=gdm
+User=xdm
+Group=xdm
 # To try to kill all clients started at greeter time or in the Init script.
 # doesn't always work, only if those clients have a window of their own
 KillInitClients=true
@@ -157,7 +157,7 @@
 # The specific gtkrc file we use.  It should be the full path to the gtkrc
 # that we need.  Unless you need a specific gtkrc that doesn't correspond to
 # a specific theme, then just use the GtkTheme key
-#GtkRC=@EXPANDED_DATADIR@/themes/Default/gtk/gtkrc
+GtkRC=@EXPANDED_DATADIR@/themes/Default/gtk-2.0/gtkrc
 
 # The GTK+ theme to use for the gui
 #GtkTheme=Default
