commit 02a48c1697a6ffe9676f20493dc426469b6f631a
Author: Marcin Banasiak <megabajt@pld-linux.org>
Date:   Thu Oct 1 21:40:09 2009 +0200

    Don't hardcode path to polkit-gnome-authentication-agent-1

diff --git a/data/greeter-autostart/Makefile.am b/data/greeter-autostart/Makefile.am
index 0d8b01a..d12246b 100644
--- a/data/greeter-autostart/Makefile.am
+++ b/data/greeter-autostart/Makefile.am
@@ -30,6 +30,7 @@ EXTRA_DIST = 			\
 	gdm-simple-greeter.desktop.in.in	\
 	gnome-power-manager.desktop.in.in	\
 	gnome-settings-daemon.desktop.in.in	\
+	polkit-gnome-authentication-agent-1.desktop.in.in \
 	$(desktop_in_files)	\
 	$(NULL)
 
@@ -39,6 +40,7 @@ DISTCLEANFILES =			\
 	gdm-simple-greeter.desktop.in	\
 	gnome-power-manager.desktop.in	\
 	gnome-settings-daemon.desktop.in	\
+	polkit-gnome-authentication-agent-1.desktop.in \
 	$(NULL)
 
 MAINTAINERCLEANFILES =                  \
diff --git a/data/greeter-autostart/polkit-gnome-authentication-agent-1.desktop.in b/data/greeter-autostart/polkit-gnome-authentication-agent-1.desktop.in
deleted file mode 100644
index b3499d1..0000000
--- a/data/greeter-autostart/polkit-gnome-authentication-agent-1.desktop.in
+++ /dev/null
@@ -1,5 +0,0 @@
-[Desktop Entry]
-Type=Application
-_Name=PolicyKit Authentication Agent
-Exec=/usr/libexec/polkit-gnome-authentication-agent-1
-OnlyShowIn=GNOME;
diff --git a/data/greeter-autostart/polkit-gnome-authentication-agent-1.desktop.in.in b/data/greeter-autostart/polkit-gnome-authentication-agent-1.desktop.in.in
new file mode 100644
index 0000000..e398cf6
--- /dev/null
+++ b/data/greeter-autostart/polkit-gnome-authentication-agent-1.desktop.in.in
@@ -0,0 +1,5 @@
+[Desktop Entry]
+Type=Application
+_Name=PolicyKit Authentication Agent
+Exec=@LIBEXECDIR@/polkit-gnome-authentication-agent-1
+OnlyShowIn=GNOME;
