--- gui/simple-greeter/gdm-user-manager.c	(wersja 6390)
+++ gui/simple-greeter/gdm-user-manager.c	(kopia robocza)
@@ -89,7 +89,7 @@ 
                                   "uucp",       \
                                   "operator",   \
                                   "nobody",     \
-                                  "gdm",        \
+                                  GDM_USERNAME, \
                                   "postgres",   \
                                   "pvm",        \
                                   "rpm",        \
--- configure.ac	(wersja 6390)
+++ configure.ac	(kopia robocza)
@@ -445,7 +445,35 @@ 
 fi
 AC_MSG_RESULT($fstype)
 
+dnl GDM user
+withval=""
+AC_ARG_WITH(user,
+            AS_HELP_STRING([--with-user=<username>],
+                           [specify gdm's user name]))
 
+if test x$withval != x; then
+	GDM_USERNAME="$withval"
+else
+	GDM_USERNAME=gdm
+fi
+AC_SUBST(GDM_USERNAME)
+AC_DEFINE_UNQUOTED([GDM_USERNAME], "$GDM_USERNAME", [User to use])
+
+dnl GDM group
+withval=""
+AC_ARG_WITH(group,
+            AS_HELP_STRING([--with-group=<username>],
+                           [specify gdm's user name]))
+
+if test x$withval != x; then
+	GDM_GROUPNAME="$withval"
+else
+	GDM_GROUPNAME=gdm
+fi
+AC_SUBST(GDM_GROUPNAME)
+AC_DEFINE_UNQUOTED([GDM_GROUPNAME], "$GDM_GROUPNAME", [Group to use])
+
+
 dnl ---------------------------------------------------------------------------
 dnl - Check for PAM stuff
 dnl ---------------------------------------------------------------------------
@@ -1415,6 +1443,7 @@ 
 gui/simple-chooser/Makefile
 gui/user-switch-applet/Makefile
 utils/Makefile
+data/gdm.conf
 data/Makefile
 data/faces/Makefile
 data/pixmaps/Makefile
@@ -1452,6 +1481,9 @@ 
 	cflags:		          ${CFLAGS}
         Maintainer mode:          ${USE_MAINTAINER_MODE}
 
+        user:                     ${GDM_USERNAME}
+        group:                    ${GDM_GROUPNAME}
+
         dbus-1 system.d dir:      ${DBUS_SYS_DIR}
         PAM prefix:               ${PAM_PREFIX}
         X server:                 ${X_SERVER}
--- daemon/gdm-display.c	(wersja 6390)
+++ daemon/gdm-display.c	(kopia robocza)
@@ -161,7 +161,7 @@ 
         g_return_val_if_fail (display->priv->access_file == NULL, FALSE);
 
         error = NULL;
-        access_file = _create_access_file_for_user (display, "gdm", &error);
+        access_file = _create_access_file_for_user (display, GDM_USERNAME, &error);
 
         if (access_file == NULL) {
                 g_critical ("could not create display access file: %s", error->message);
--- daemon/gdm-chooser-server.c	(wersja 6390)
+++ daemon/gdm-chooser-server.c	(kopia robocza)
@@ -552,14 +552,14 @@ 
                                          g_param_spec_string ("user-name",
                                                               "user name",
                                                               "user name",
-                                                              "gdm",
+                                                              GDM_USERNAME,
                                                               G_PARAM_READWRITE | G_PARAM_CONSTRUCT));
         g_object_class_install_property (object_class,
                                          PROP_GROUP_NAME,
                                          g_param_spec_string ("group-name",
                                                               "group name",
                                                               "group name",
-                                                              "gdm",
+                                                              GDM_GROUPNAME,
                                                               G_PARAM_READWRITE | G_PARAM_CONSTRUCT));
         signals [HOSTNAME_SELECTED] =
                 g_signal_new ("hostname-selected",
--- daemon/gdm-welcome-session.c	(wersja 6390)
+++ daemon/gdm-welcome-session.c	(kopia robocza)
@@ -1122,14 +1122,14 @@ 
                                          g_param_spec_string ("user-name",
                                                               "user name",
                                                               "user name",
-                                                              "gdm",
+                                                              GDM_USERNAME,
                                                               G_PARAM_READWRITE | G_PARAM_CONSTRUCT));
         g_object_class_install_property (object_class,
                                          PROP_GROUP_NAME,
                                          g_param_spec_string ("group-name",
                                                               "group name",
                                                               "group name",
-                                                              "gdm",
+                                                              GDM_GROUPNAME,
                                                               G_PARAM_READWRITE | G_PARAM_CONSTRUCT));
         g_object_class_install_property (object_class,
                                          PROP_SERVER_ADDRESS,
--- daemon/gdm-factory-slave.c	(wersja 6390)
+++ daemon/gdm-factory-slave.c	(kopia robocza)
@@ -489,7 +489,7 @@ 
         /* FIXME: send a signal back to the master */
 
         /* Run the init script. gdmslave suspends until script has terminated */
-        gdm_slave_run_script (GDM_SLAVE (slave), GDMCONFDIR "/Init", "gdm");
+        gdm_slave_run_script (GDM_SLAVE (slave), GDMCONFDIR "/Init", GDM_USERNAME);
 
         slave->priv->greeter_server = gdm_greeter_server_new (display_id);
         g_signal_connect (slave->priv->greeter_server,
--- daemon/gdm-greeter-server.c	(wersja 6390)
+++ daemon/gdm-greeter-server.c	(kopia robocza)
@@ -1113,14 +1113,14 @@ 
                                          g_param_spec_string ("user-name",
                                                               "user name",
                                                               "user name",
-                                                              "gdm",
+                                                              GDM_USERNAME,
                                                               G_PARAM_READWRITE | G_PARAM_CONSTRUCT));
         g_object_class_install_property (object_class,
                                          PROP_GROUP_NAME,
                                          g_param_spec_string ("group-name",
                                                               "group name",
                                                               "group name",
-                                                              "gdm",
+                                                              GDM_GROUPNAME,
                                                               G_PARAM_READWRITE | G_PARAM_CONSTRUCT));
         signals [BEGIN_VERIFICATION] =
                 g_signal_new ("begin-verification",
--- daemon/gdm-simple-slave.c	(wersja 6390)
+++ daemon/gdm-simple-slave.c	(kopia robocza)
@@ -367,7 +367,7 @@ 
         queue_start_session (slave);
 
         /* Run the PostLogin script. gdmslave suspends until script has terminated */
-        gdm_slave_run_script (GDM_SLAVE (slave), GDMCONFDIR "/PostLogin", "gdm");
+        gdm_slave_run_script (GDM_SLAVE (slave), GDMCONFDIR "/PostLogin", GDM_USERNAME);
 }
 
 static void
@@ -855,7 +855,7 @@ 
         }
 
         /* Run the init script. gdmslave suspends until script has terminated */
-        gdm_slave_run_script (GDM_SLAVE (slave), GDMCONFDIR "/Init", "gdm");
+        gdm_slave_run_script (GDM_SLAVE (slave), GDMCONFDIR "/Init", GDM_USERNAME);
 
         create_new_session (slave);
 
--- daemon/gdm-xdmcp-chooser-slave.c	(wersja 6390)
+++ daemon/gdm-xdmcp-chooser-slave.c	(kopia robocza)
@@ -190,7 +190,7 @@ 
         }
 
         /* Run the init script. gdmslave suspends until script has terminated */
-        gdm_slave_run_script (GDM_SLAVE (slave), GDMCONFDIR "/Init", "gdm");
+        gdm_slave_run_script (GDM_SLAVE (slave), GDMCONFDIR "/Init", GDM_USERNAME);
 
         slave->priv->chooser_server = gdm_chooser_server_new (display_id);
         g_signal_connect (slave->priv->chooser_server,
--- data/gdm.conf	(wersja 6390)
+++ data/gdm.conf	(kopia robocza)
@@ -1,60 +0,0 @@ 
-<!DOCTYPE busconfig PUBLIC
- "-//freedesktop//DTD D-BUS Bus Configuration 1.0//EN"
- "http://www.freedesktop.org/standards/dbus/1.0/busconfig.dtd">
-<busconfig>
-
-  <!-- Only root can own the service -->
-  <policy user="root">
-    <allow own="org.gnome.DisplayManager"/>
-
-    <allow send_interface="org.gnome.DisplayManager.Manager"/>
-    <allow send_interface="org.gnome.DisplayManager.Display"/>
-    <allow send_interface="org.gnome.DisplayManager.LocalDisplayFactory"/>
-    <allow send_interface="org.gnome.DisplayManager.Settings"/>
-    <allow send_interface="org.gnome.DisplayManager.Slave"/>
-    <allow send_interface="org.gnome.DisplayManager.Session"/>
-    <allow send_destination="org.gnome.DisplayManager"
-           send_interface="org.gnome.DBus.Properties" />
-  </policy>
-
-  <policy context="default">
-    <deny send_interface="org.gnome.DisplayManager.Manager"/>
-    <deny send_interface="org.gnome.DisplayManager.Display"/>
-    <deny send_interface="org.gnome.DisplayManager.LocalDisplayFactory"/>
-    <deny send_interface="org.gnome.DisplayManager.Settings"/>
-    <deny send_interface="org.gnome.DisplayManager.Slave"/>
-    <deny send_interface="org.gnome.DisplayManager.Session"/>
-    <deny send_destination="org.gnome.DisplayManager"
-          send_interface="org.gnome.DBus.Properties" />
-
-    <allow send_interface="org.gnome.DisplayManager.Display"
-           send_member="GetId"/>
-    <allow send_interface="org.gnome.DisplayManager.Display"
-           send_member="GetRemoteHostname"/>
-    <allow send_interface="org.gnome.DisplayManager.Display"
-           send_member="GetSeatId"/>
-    <allow send_interface="org.gnome.DisplayManager.Display"
-           send_member="GetX11DisplayName"/>
-    <allow send_interface="org.gnome.DisplayManager.Display"
-           send_member="GetX11DisplayNumber"/>
-    <allow send_interface="org.gnome.DisplayManager.Display"
-           send_member="IsLocal"/>
-
-    <allow send_interface="org.gnome.DisplayManager.LocalDisplayFactory"
-           send_member="CreateTransientDisplay"/>
-
-    <allow send_interface="org.gnome.DisplayManager.Manager"
-           send_member="GetDisplays"/>
-
-  </policy>
-
-  <policy user="gdm">
-    <allow send_interface="org.gnome.DisplayManager.Manager"/>
-    <allow send_interface="org.gnome.DisplayManager.Display"/>
-    <allow send_interface="org.gnome.DisplayManager.LocalDisplayFactory"/>
-    <allow send_interface="org.gnome.DisplayManager.Settings"/>
-    <allow send_destination="org.gnome.DisplayManager"
-           send_interface="org.gnome.DBus.Properties" />
-  </policy>
-
-</busconfig>
--- data/gdm.conf.in	(wersja 0)
+++ data/gdm.conf.in	(wersja 0)
@@ -0,0 +1,60 @@ 
+<!DOCTYPE busconfig PUBLIC
+ "-//freedesktop//DTD D-BUS Bus Configuration 1.0//EN"
+ "http://www.freedesktop.org/standards/dbus/1.0/busconfig.dtd">
+<busconfig>
+
+  <!-- Only root can own the service -->
+  <policy user="root">
+    <allow own="org.gnome.DisplayManager"/>
+
+    <allow send_interface="org.gnome.DisplayManager.Manager"/>
+    <allow send_interface="org.gnome.DisplayManager.Display"/>
+    <allow send_interface="org.gnome.DisplayManager.LocalDisplayFactory"/>
+    <allow send_interface="org.gnome.DisplayManager.Settings"/>
+    <allow send_interface="org.gnome.DisplayManager.Slave"/>
+    <allow send_interface="org.gnome.DisplayManager.Session"/>
+    <allow send_destination="org.gnome.DisplayManager"
+           send_interface="org.gnome.DBus.Properties" />
+  </policy>
+
+  <policy context="default">
+    <deny send_interface="org.gnome.DisplayManager.Manager"/>
+    <deny send_interface="org.gnome.DisplayManager.Display"/>
+    <deny send_interface="org.gnome.DisplayManager.LocalDisplayFactory"/>
+    <deny send_interface="org.gnome.DisplayManager.Settings"/>
+    <deny send_interface="org.gnome.DisplayManager.Slave"/>
+    <deny send_interface="org.gnome.DisplayManager.Session"/>
+    <deny send_destination="org.gnome.DisplayManager"
+          send_interface="org.gnome.DBus.Properties" />
+
+    <allow send_interface="org.gnome.DisplayManager.Display"
+           send_member="GetId"/>
+    <allow send_interface="org.gnome.DisplayManager.Display"
+           send_member="GetRemoteHostname"/>
+    <allow send_interface="org.gnome.DisplayManager.Display"
+           send_member="GetSeatId"/>
+    <allow send_interface="org.gnome.DisplayManager.Display"
+           send_member="GetX11DisplayName"/>
+    <allow send_interface="org.gnome.DisplayManager.Display"
+           send_member="GetX11DisplayNumber"/>
+    <allow send_interface="org.gnome.DisplayManager.Display"
+           send_member="IsLocal"/>
+
+    <allow send_interface="org.gnome.DisplayManager.LocalDisplayFactory"
+           send_member="CreateTransientDisplay"/>
+
+    <allow send_interface="org.gnome.DisplayManager.Manager"
+           send_member="GetDisplays"/>
+
+  </policy>
+
+  <policy user="@GDM_USERNAME@">
+    <allow send_interface="org.gnome.DisplayManager.Manager"/>
+    <allow send_interface="org.gnome.DisplayManager.Display"/>
+    <allow send_interface="org.gnome.DisplayManager.LocalDisplayFactory"/>
+    <allow send_interface="org.gnome.DisplayManager.Settings"/>
+    <allow send_destination="org.gnome.DisplayManager"
+           send_interface="org.gnome.DBus.Properties" />
+  </policy>
+
+</busconfig>
--- data/Makefile.am	(wersja 6390)
+++ data/Makefile.am	(kopia robocza)
@@ -34,7 +34,8 @@ 
 		<$(srcdir)/gdm.conf-custom.in >gdm.conf-custom
 
 dbusconfdir = $(DBUS_SYS_DIR)
-dbusconf_DATA = gdm.conf
+dbusconf_in_files = gdm.conf.in
+dbusconf_DATA = $(dbusconf_in_files:.conf.in=.conf)
 
 @INTLTOOL_SCHEMAS_RULE@
 
@@ -72,7 +73,7 @@ 
 EXTRA_DIST =			\
 	$(schemas_in_files)	\
 	$(schemas_DATA)		\
-	$(dbusconf_DATA)	\
+	$(dbusconf_in_files)	\
 	gdm.schemas.in.in	\
 	gdm.conf-custom.in 	\
 	Xsession.in 		\
@@ -95,6 +96,7 @@ 
 	$(NULL)
 
 DISTCLEANFILES =			\
+	$(dbusconf_DATA)			\
 	gdm.schemas			\
 	$(NULL)
 
--- daemon/gdm-display-access-file.c~	2008-07-31 02:19:06.000000000 +0200
+++ daemon/gdm-display-access-file.c	2008-08-20 10:51:12.092856608 +0200
@@ -254,7 +254,7 @@
                 }
 
                 g_chmod (GDM_XAUTH_DIR, S_ISVTX | S_IRWXU | S_IRWXG | S_IROTH | S_IXOTH);
-                _get_uid_and_gid_for_user ("gdm", &uid, &gid);
+                _get_uid_and_gid_for_user (GDM_USERNAME, &uid, &gid);
                 if (chown (GDM_XAUTH_DIR, 0, gid) != 0) {
                         g_warning ("Unable to change owner of '%s'",
                                    GDM_XAUTH_DIR);
--- data/gdm.schemas.in.in~	2008-07-03 21:57:25.000000000 +0200
+++ data/gdm.schemas.in.in	2008-08-20 10:59:55.120863811 +0200
@@ -4,12 +4,12 @@
     <schema>
       <key>daemon/User</key>
       <signature>s</signature>
-      <default>gdm</default>
+      <default>@GDM_USERNAME@</default>
     </schema>
     <schema>
       <key>daemon/Group</key>
       <signature>s</signature>
-      <default>gdm</default>
+      <default>@GDM_GROUPNAME@</default>
     </schema>
     <schema>
       <key>daemon/TimedLoginEnable</key>
--- data/Makefile.am~	2008-08-20 11:00:48.000000000 +0200
+++ data/Makefile.am	2008-08-20 11:04:27.274257897 +0200
@@ -47,6 +47,8 @@
 	sed	-e 's,[@]GDMPREFETCHCMD[@],$(GDMPREFETCHCMD),g' \
 		-e 's,[@]GDM_CUSTOM_CONF[@],$(GDM_CUSTOM_CONF),g' \
 		-e 's,[@]GDM_USER_PATH[@],$(GDM_USER_PATH),g' \
+		-e 's,[@]GDM_USERNAME[@],$(GDM_USERNAME),g' \
+		-e 's,[@]GDM_GROUPNAME[@],$(GDM_GROUPNAME),g' \
 		-e 's,[@]HALT_COMMAND[@],$(HALT_COMMAND),g' \
 		-e 's,[@]REBOOT_COMMAND[@],$(REBOOT_COMMAND),g' \
 		-e 's,[@]SOUND_PROGRAM[@],$(SOUND_PROGRAM),g' \
