diff -urN gdm-2.6.0.3.orig/config/Xsession.in gdm-2.6.0.3/config/Xsession.in
--- gdm-2.6.0.3.orig/config/Xsession.in	2004-05-28 21:50:17.000000000 +0200
+++ gdm-2.6.0.3/config/Xsession.in	2004-07-09 15:07:16.039478016 +0200
@@ -35,6 +35,10 @@
 test -f /etc/xprofile && . /etc/xprofile
 test -f "$HOME/.xprofile" && . "$HOME/.xprofile"
 
+if [ ! -f $HOME/.profile ] && echo $SHELL |grep -q "bash"; then
+    test -f $HOME/.bash_profile && . $HOME/.bash_profile
+fi
+
 # Translation stuff
 if [ -x "@EXPANDED_LIBEXECDIR@/gdmtranslate" ] ; then
   gdmtranslate="@EXPANDED_LIBEXECDIR@/gdmtranslate"
@@ -222,6 +226,23 @@
   fi
 fi
 
+# Keep in sync with Xclients from xinitrc-ng 
+syswmfile="/etc/sysconfig/desktop"
+
+# In this file "$wmstyledir" value may be given
+[ -f "$syswmfile" ] && . $syswmfile
+
+if [ -z "$wmstyledir" ]; then
+    # Keep in sync with Xclients from xinitrc-ng 
+    wmstyledir="/etc/sysconfig/wmstyle"
+fi
+
+if [ -x "$HOME/bin/$command.sh" ]; then
+    command="$HOME/bin/$command.sh"
+elif [ -x "$wmstyledir/$command.sh" ]; then
+    command="$wmstyledir/$command.sh"
+fi
+
 # add ssh-agent if found
 sshagent="`gdmwhich ssh-agent`"
 if [ -n "$sshagent" ] && [ -x "$sshagent" ] && [ -z "$SSH_AUTH_SOCK" ]; then
