--- gdm-2.0beta4/daemon/slave.c.rhlang	Tue Oct  3 13:26:23 2000
+++ gdm-2.0beta4/daemon/slave.c	Tue Oct  3 13:32:50 2000
@@ -374,10 +374,13 @@
 	setenv ("PATH", GdmDefaultPath, TRUE);
 
     /* Set locale */
-    if (!strcasecmp (language, "english"))
-	setenv ("LANG", "C", TRUE);
-    else
-	setenv ("LANG", language, TRUE);
+    if (!strcasecmp (language, "english")) {
+	setenv ("LANG",     "C", TRUE);
+	setenv ("GDM_LANG", "C", TRUE);
+    } else {
+	setenv ("LANG",     language, TRUE);
+	setenv ("GDM_LANG", language, TRUE);
+    }
     
     /* If script fails reset X server and restart greeter */
     if (gdm_exec_script (d, GdmPreSession) != EXIT_SUCCESS) 
