diff -urN gdm-2.20.0/configure.ac gdm-2.20.0.new/configure.ac
--- gdm-2.20.0/configure.ac	2007-09-17 16:46:22.000000000 +0200
+++ gdm-2.20.0.new/configure.ac	2007-09-21 17:29:32.000000000 +0200
@@ -1206,6 +1201,10 @@
    X_SERVER="/usr/X11R6/bin/X"
    X_CONFIG_OPTIONS="-audit 0"
 fi
+X_PATH="/usr/bin"
+X_SERVER_PATH="/usr/bin"
+X_SERVER="/usr/bin/Xorg vt9"
+X_CONFIG_OPTIONS="-audit 0"
 
 dnl ---------------------------------------------------------------------------
 dnl - Check for Xnest / Xephyr support
