--- gdm-2.0beta4/gui/gdmlogin.c~	Wed Sep 29 18:56:35 1999
+++ gdm-2.0beta4/gui/gdmlogin.c	Tue May 23 09:12:35 2000
@@ -461,7 +461,7 @@
     gtk_widget_set_sensitive (GTK_WIDGET (sessmenu), FALSE);
 
     /* Previously saved session not found in ~user/.gnome/gdm */
-    if (! strlen (savedsess)) {
+    if (!(savedsess && strcmp ("(null)", savedsess) && strlen (savedsess))) {
 	/* If "Last" is chosen run Default, else run user's current selection */
 	if (!strcasecmp (cursess, lastsess))
 	    session = defsess;
