diff -aurN gdm-2.14.1.orig/config/Makefile.am gdm-2.14.1/config/Makefile.am
--- gdm-2.14.1.orig/config/Makefile.am	2006-03-29 01:43:20.000000000 +0200
+++ gdm-2.14.1/config/Makefile.am	2006-04-12 17:38:22.703380096 +0200
@@ -5,7 +5,7 @@
 instsessdir = $(datadir)/xsessions
 initdir = $(gdmconfdir)/Init
 authdir = $(localstatedir)/gdm
-logdir = $(localstatedir)/log/gdm
+logdir = /var/log/gdm
 gnomercdir = $(gdmconfdir)
 postdir = $(gdmconfdir)/PostSession
 predir = $(gdmconfdir)/PreSession
diff -aurN gdm-2.14.1.orig/configure.ac gdm-2.14.1/configure.ac
--- gdm-2.14.1.orig/configure.ac	2006-04-10 22:08:15.000000000 +0200
+++ gdm-2.14.1/configure.ac	2006-04-12 17:39:15.822304792 +0200
@@ -793,7 +793,7 @@
 #
 AC_SUBST(authdir, ${localstatedir}/gdm)
 AC_SUBST(gdmlocaledir, ${gdmconfdir})
-AC_SUBST(logdir, ${localstatedir}/log/gdm)
+AC_SUBST(logdir, /var/log/gdm)
 AC_SUBST(pixmapdir, ${datadir}/pixmaps)
 
 withval=""
