--- gdm-2.0beta4/daemon/slave.c	Mon Jul 17 13:37:05 2000
+++ gdm-2.0beta4/daemon/slave.c~	Mon Jul 17 13:15:48 2000
@@ -456,7 +456,7 @@
 	shell = gdm_get_user_shell ();
 	shell_hyphen = g_strconcat ("-", shell, NULL);
 
-	execl (shell, shell_hyphen, sesspath, NULL);
+	execl (shell, "-", "-c", sesspath, NULL);
 	
 	gdm_error (_("gdm_slave_session_start: Could not start session `%s'"), sesspath);
 	
