--- gdm-2.24.0.orig/daemon/gdm-server.c	2008-10-03 00:53:12.000000000 +0200
+++ gdm-2.24.0/daemon/gdm-server.c	2008-10-03 00:53:57.000000000 +0200
@@ -419,7 +419,7 @@
         g_debug ("GdmServer: Opening logfile for server %s", log_path);
 
         VE_IGNORE_EINTR (g_unlink (log_path));
-        VE_IGNORE_EINTR (logfd = open (log_path, O_CREAT|O_TRUNC|O_WRONLY|O_EXCL, 0644));
+        VE_IGNORE_EINTR (logfd = open (log_path, O_CREAT|O_TRUNC|O_WRONLY|O_EXCL, 0640));
 
         g_free (log_path);
 
