diff -aurN gdm-2.13.0.7.orig/gui/gdmphotosetup.desktop.in gdm-2.13.0.7/gui/gdmphotosetup.desktop.in
--- gdm-2.13.0.7.orig/gui/gdmphotosetup.desktop.in	2003-07-26 20:57:15.000000000 +0200
+++ gdm-2.13.0.7/gui/gdmphotosetup.desktop.in	2006-02-05 17:22:55.237522000 +0100
@@ -9,7 +9,7 @@
 Terminal=false
 Type=Application
 NotShowIn=GNOME;
-Categories=GNOME;GTK;Settings;
+Categories=GTK;GNOME;Settings;X-Capplet;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gdm
 X-GNOME-Bugzilla-Component=general
